ext/standard/basic_functions.stub.php

```diff
@@ -153,11 +153,11 @@ function array_merge(array ...$arrays): array {}
 
 function array_merge_recursive(array ...$arrays): array {}
 
-function array_replace(array $array1, array ...$arrays): array {}
+function array_replace(array $array, array ...$substitutions): array {}
 
-function array_replace_recursive(array $array1, array ...$arrays): array {}
+function array_replace_recursive(array $array, array ...$substitutions): array {}
```

Comment:
How about `$replacements`? It would be consistent with `array_splice()` and heading towards consistency with many other `$replace` parameters.

Reply:
Looking through the stub file, `$replacement` is only used once, in `array_splice()`.  `$replace` is used a whole bunch of times.  `$substitutions` is used nowhere but in this change.  So, yeah, $replacements makes sense.

I think we should then also change `array_splice()` to `$replacements`.  It's not variadic, but it is an array.